as-controller-board-status/functions/output.go

```diff
@@ -262,16 +260,11 @@ func (boardStatus *CheckBoardStatus) processVendingDoorState(lc logger.LoggingCl
 	return nil
 }
 
-// GetStatus is a REST API endpoint that enables a web UI or some other downstream
-// service to inquire about the status of the upstream Automated Checkout hardware interface(s).
-func GetStatus(writer http.ResponseWriter, req *http.Request) {
-	utilities.ProcessCORS(writer, req, func(writer http.ResponseWriter, req *http.Request) {
-		controllerBoardStatusJSON, err := utilities.GetAsJSON(controllerBoardStatus)
-		if err != nil {
-			utilities.WriteStringHTTPResponse(writer, req, http.StatusInternalServerError, "Failed to serialize the controller board's current state.", true)
-			return
-		}
+func (boardStatus *CheckBoardStatus) GetControllerBoardStatus() ControllerBoardStatus {
+	return controllerBoardStatus
+}
```

Comment:
Is this necessary?  if controllerBoardStatus is a global variable, why could we directly access it?

Reply:
Because it is global to this package, not the others...

Reply:
controllerBoardStatus is not a global variable

Reply:
local globals like `controllerBoardStatus` are bad practice.
We should move `controllerBoardStatus ` into the `CheckBoardStatus` struct and not have it as package global.

Reply:
I would like to have discussion about this please

Reply:
issue #46 created to track this change to be done